GenderPayGap.WebUI/Views/Viewing/Finder/Parts/Facets/Keywords.cshtml

```diff
@@ -1,20 +1,17 @@
-﻿@model GenderPayGap.WebUI.Models.Search.SearchViewModel
+@model GenderPayGap.WebUI.Models.Search.SearchViewModel
 <div id="SearchByKeywordsGroup" class="form-group" style="position: relative;">
-
+    <h1 class="heading-large">
+        Search by <span class="visuallyhidden">either employer name or SIC code</span>
+    </h1>
     <fieldset class="inline">
-        <legend class="visually-hidden">Radiobutton selecting a search term</legend>
-        <label for="FinderKeywords">
-            <h1 class="heading-large">
-                Search by <span class="visuallyhidden">“@(Model.search)”</span>
-            </h1>
-        </label>
-
+        <legend class="visually-hidden">Radiobuttons selecting either employer name or SIC code</legend>

```

Comment:
Perhaps `Select how you want to search: either by employer name or by Sector (SIC code)`